helix-core/src/test/java/org/apache/helix/integration/task/TestTaskSchedulingTwoCurrentStates.java

```diff
@@ -138,6 +139,18 @@ public void testTargetedTaskTwoCurrentStates() throws Exception {
     JobQueue.Builder jobQueue = TaskTestUtil.buildJobQueue(jobQueueName);
     jobQueue.enqueueJob("JOB0", jobBuilder0);
 
+    // Make sure master has been correctly switched to Participant1
+    boolean isMasterSwitchedToCorrectInstance = TestHelper.verify(() -> {
+      ExternalView externalView = _gSetupTool.getClusterManagementTool().getResourceExternalView(CLUSTER_NAME, DATABASE);
+      Map <String, String> stateMap = externalView.getStateMap(DATABASE + "_0");
+      String state = stateMap.getOrDefault(PARTICIPANT_PREFIX + "_" + (_startPort + 1), null);
+      if (state == null) {
+        return false;
+      }
```

Comment:
If you're expecting null, there's really no reason to use `getOrDefault`?

Reply:
In this case, a simple containsKey check would do..

Reply:
Fixed. Replaced with containsKey